src/DynamoCoreWpf/Views/PackageManager/PackagePathView.xaml

```diff
@@ -17,14 +17,14 @@
         d:DesignWidth="300"
         Background="#515151"
         Style="{DynamicResource DynamoWindowStyle}"
-        d:DataContext="{d:DesignInstance ViewModels:InstalledPackagesViewModel, IsDesignTimeCreatable=False}">
+        d:DataContext="{d:DesignInstance ViewModels:PackagePathViewModel, IsDesignTimeCreatable=False}">
```

Comment:
Was this a bug before?

Reply:
Yes, copy/paste error. Looks like it didn't matter but I was seeing compile time warnings.

Reply:
so is the datacontext being reset correctly somewhere else?